hack/common.sh

```diff
@@ -42,9 +42,14 @@ default_local_storage_image="quay.io/gnufied/local-registry:v4.2.0"
 LOCAL_STORAGE_IMAGE_NAME="${LOCAL_STORAGE_IMAGE:-$default_local_storage_image}"
 
 OCS_CLUSTER_UNINSTALL="${OCS_CLUSTER_UNINSTALL:-true}"
+OCS_SUBSCRIPTION_CHANNEL=${OCS_SUBSCRIPTION_CHANNEL:-alpha}
 MUST_GATHER_IMAGE_NAME="ocs-must-gather"
 MUST_GATHER_FULL_IMAGE_NAME="${IMAGE_REGISTRY}/${REGISTRY_NAMESPACE}/${MUST_GATHER_IMAGE_NAME}:${IMAGE_TAG}"
 
+UPGRADE_FROM_OCS_REGISTRY_IMAGE="${UPGRADE_FROM_OCS_REGISTRY_IMAGE:-quay.io/ocs-dev/ocs-registry:4.2.0}"
```

Comment:
Using quay.io/ocs-dev/ocs-registry:4.2.0 as the UPGRADE_FROM image. This is really the simplest solution that works, and it makes sense as this is our first release. What do you think @kshlm ?

Reply:
Well, there really isn't any other way right now. ocs-registry:4.2.0 is the image to be used.